src/main/java/ua/softserve/rv_028/issuecitymonitor/dto/IssueLocationDto.java

```diff
@@ -0,0 +1,97 @@
+package ua.softserve.rv_028.issuecitymonitor.dto;
+
+import ua.softserve.rv_028.issuecitymonitor.entity.Issue;
+import ua.softserve.rv_028.issuecitymonitor.entity.enums.IssueCategory;
+
+public class IssueLocationDto {
+
+    private Long id;
+    private String title;
+    private String description;
+    private String firstname;
+    private String lastname;
+    private double latitude;
+    private double longitude;
+
+    public IssueLocationDto() {}
+
+
+    public IssueLocationDto(Long id, UserDto userDto, String title, String description, String initialDate,
```

Comment:
When we would have a lot of issues, this will make a content load problem. Just simplify this object to latitude, longitude, id fields.